### ISSUE TITLE
password recovery

### ISSUE THREAD

**Issue by bksubhuti** [dhammacakka/pm12e#25]
Is there a "free" password recovery routine?
Many forget their password.

This issue is a low priority and NO priority if there is no django routine for this.

**Comment by sumbodhi** [comment]
There is a Django routine but it requires a valid email so that Django sends you a password reset link.

**Comment by bksubhuti** [comment]
yes.. i think that most give email addresses on signup.  some in myanmar don't have email..just facebook and phone.
